tests/objvariant/tcheckedfield1.nim

```diff
@@ -1,5 +1,5 @@
 discard """
-  nimout: "Warning: cannot prove that field 'x.s' is accessible [ProveField]"
+  nimout: "tcheckedfield1.nim(40, 6) Warning: cannot prove that field 'x.s' is accessible [ProveField]"
   line:51
```

Comment:
another bug: if i change this to `line:1234`, the test still passes

Reply:
where is the logic?

Reply:
pre-existing, it's unrelated to your PR, just that i noticed it while reviewing your PR